### PR TITLE
[Modular] Adds a thinner variant of the choker

### PR DESCRIPTION
**Pull request by Jureiia**
## About The Pull Request
I, and some people I brought it up with agreed that they thought the current choker was way too thick for what it was supposed to be, so I created an alternate version called the "Thin Choker" and put it in all the appropriate places (loadout and lustwish). I chose to make it another item just in case some people would prefer the original version and I don't want to take that away from them.

## How This Contributes To The Skyrat Roleplay Experience
More options!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

vcLiTwI7yJ
20pgTdDI5n
ZiSTvSe9LV
dreamseeker_7tOQKjYHSM
vdPRTB8Sjf
dreamseeker_9LL0ahu1dw

</details>

## Changelog

:cl:
add: A new thinner choker variation, now available in the loadout and lustwish vendor.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/neck/collars.dm [hunk 1]
```diff
@@ -52,6 +52,14 @@
 	greyscale_config_worn = /datum/greyscale_config/collar/choker/worn
 	greyscale_colors = "#222222"
 
+/obj/item/clothing/neck/human_petcollar/thinchoker
+	name = "thin choker"
+	desc = "Like the thicker one, but just a bit thinner!"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/neck/collars.dm]: I think this is a bit funnier and catchier.
```suggestion
	desc = "Like the normal one, but thinner!"
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/modules/clothing/neck/collars.dm (hunk 1 above)
Comment (Jureiia) [reply modular_skyrat/modules/customization/modules/clothing/neck/collars.dm]: Hell yeah!

**Comment by Jureiia** [comment]
why break >:(

**Review by GoldenAlpharex — APPROVED** [review]
Don't worry about the failure, it's unrelated to what you've done!

**Pull request merged**